libraries/plugins/rc/resource_user.cpp

```diff
@@ -0,0 +1,30 @@
+
+#include <steem/plugins/rc/resource_user.hpp>
+
+#include <steem/protocol/transaction.hpp>
+
+namespace steem { namespace plugins { namespace rc {
+
+using namespace steem::protocol;
+
+account_name_type get_resource_user( const signed_transaction& tx )
```

Comment:
This is a departure from how bandwidth is charged today. Subtle to most users, but the decision should be purposeful.

This method returns a single account to charge bandwidth to where the current algorithm charges it to all parties involved in the transaction.